packages/fixtures/boards/usbvil/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "skipSnapshot": true,
+  "layers": [
+    {
+      "name": "PIC18F14K50.GBL",
```

Comment:
Can these files be renamed to have lower-case extensions for consistency? Just wanna avoid any weird case-sensitive file-system issues in any refactors down the road

Reply:
But shouldn't we have some test cases with uppercase extensions to be sure not to break that behaviour?

Reply:
Yup! And we do in the normal `whats-that-gerber` unit tests (`packages/fixtures/gerber-filenames.json`). Better to test that functionality there than in integration tests